app/models/feedback_message.rb

```diff
@@ -13,4 +13,6 @@
 
 class FeedbackMessage < ApplicationRecord
   belongs_to :user
+
+  validates :message, presence: true, length: { minimum: 10, maximum: 200 }
```

Comment:
```suggestion
   validates :message, presence: true
```
Sometimes the feedback messages are longer than that, and that's OK. I don't think there are existing data constraints on this.

Reply:
Ok, thanks! I'll remove this.